bindings/wasm/examples/vite.config.js

```diff
@@ -3,7 +3,7 @@ import {resolve} from 'path'
 import {defineConfig} from 'vite'
 
 export default defineConfig({
-  test: {testTimeout: 15000},
+  test: {timeout: 10000},
```

Comment:
It's timing out because of a rebase issue - this is the wrong key, so we're using the default of 5000.

Reply:
Ahhh ok, I thought I picked the right one on that by obviously I was going a bit crosseyed going through the autoformat infected diffs (my own fault of course).